### DISCUSSION TITLE
Trying to get Copilot to work in VSCode

### DISCUSSION THREAD

**Discussion by J534** [community/community#65279] [Copilot Conversations]
### Select Topic Area

General

### Body

Trying to get github copilot working in VSCode. I've installed the extension and in the lower right corner of the vscode window it says:

Signing in to github.com...
Source: Github Authentication (extension)

Just sits there loading and doesn't do anything. Do any of you know how I can get it working? 

Thanks. 

I'm currently using SteamOS which is a flavor or arch linux.

**Comment by arafajs** [comment]
This error indicates that you either do not have a GitHub Copilot subscription, or there was an error connecting to the GitHub API to request a token to use GitHub Copilot.
